src/ansys/fluent/core/session_pure_meshing.py

```diff
@@ -147,3 +147,45 @@ def transfer_mesh_to_solvers(
             clean_up_mesh_file,
             overwrite_previous,
         )
+
+    def read_case(
+        self,
+        file_name: str,
+    ):
+        """Reads a case file.
+
+        Parameters
+        ----------
+        file_name : str
+            Case file name
+        """
+        if pypim.is_configured():
+            self._pypim_upload(
+                file_name=file_name,
+                on_uploaded=self.tui.file.read_case,
```

Comment:
Updated `on_uploaded = (lambda file_names: self.tui.file.read_case(file_names) -> self.tui.file.read_case` (for meshing)

Reply:
This looks good 👍